contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -0,0 +1,276 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import {RouterBase, ITypeAndVersion} from "./RouterBase.sol";
+import {IFunctionsRouter} from "./interfaces/IFunctionsRouter.sol";
+import {IFunctionsCoordinator} from "./interfaces/IFunctionsCoordinator.sol";
+import {IFunctionsSubscriptions, FunctionsSubscriptions, IFunctionsBilling} from "./FunctionsSubscriptions.sol";
+
+contract FunctionsRouter is RouterBase, IFunctionsRouter, FunctionsSubscriptions {
+  event RequestStart(
```

Comment:
1. I suggest to make `donId` and `subscriptionId` indexed args.
2. Can we add these to the event?

```
  address requestingContract,
  address requestInitiator,
  address subscriptionOwner,
```

Reply:
The complete event sig:

```
event RequestStart(
  bytes32 indexed requestId,
  bytes32 indexed donId,
  uint64 indexed subscriptionId,
  address subscriptionOwner,
  address requestingContract,
  address requestInitiator,
  bytes calldata data,
  uint16 dataVersion,
  uint32 callbackGasLimit
);
```